### ISSUE TITLE
Configruable clock

### ISSUE THREAD

**Issue by sumnerevans** [ColoradoSchoolOfMines/visplay#11]
With Visplay on the ALAMODE TVs, I want to be able to show a clock because the one in ALAMODE is broken. This should be highly configurable (position, font, margin, border, color, overlay, transparent, on/off, timezone, etc.).

**Comment by jackrosenthal** [comment]
xdaliclock?

**Comment by Warfields** [comment]
This where things might get interesting, do we allow for the execution of user scripts and code? Like if someone has a cool clock!

**Comment by robozman** [comment]
If we use xembed then embedding arbitrary x applications should be possible. Somewhat out of scope for now, but a good thought to keep in mind for down the road.

**Comment by sumnerevans** [comment]
@robozman, agree. I mean, we are expecting to need to re-architect this eventually anyway, and dealing with this issue should be part of that process.

**Comment by robozman** [comment]
Now that we are using qt, we can display arbitrary javascript. The possibilities are endless.

**Comment by jackrosenthal** [comment]
There's a JS version too: https://www.jwz.org/xdaliclock/
